app/controllers/dashboard_controller.rb

```diff
@@ -43,6 +43,7 @@ def index
   #Owner Dashboard - start project
   #other methods in AddWorkHelper
   def startproject
+    @work = Work.new
```

Comment:
I don't understand why we need a new work created here.  Isn't it done under the create_work under the add_work_helper?